crates/infra/cli/src/commands/lint/mod.rs

```diff
@@ -38,6 +33,8 @@ enum LintCommand {
     MarkdownLinkCheck,
     /// Check for violations in Markdown files.
     MarkdownLint,
+    /// Format all Rust source files.
+    Rustfmt,
```

Comment:
nitpick: It's actually cargo-fmt command that we're executing (a wrapper, but still), not rustfmt, which can only format a single file.

It's not important but I just found the rename confusing, given the "command" in the enum name and the description saying "all source files".

Reply:
I stand corrected; I don't know why I thought `rustfmt` can only format a single file. My bad! Looks good! :+1: